### PR TITLE
Implement Create Wallet Account Structure (But EOS Disabled)

### PR DESCRIPTION
**Pull request by kylanhurt**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/actions/CreateWalletActions.js [hunk 1]
```diff
@@ -45,15 +50,124 @@ export const createCurrencyWallet = (
...
 }
+
+export const fetchAccountActivationInfo = (currencyCode: string) => async (dispatch: Dispatch, getState: GetState) => {
+  const state = getState()
+  const account = CORE_SELECTORS.getAccount(state)
+  const currencyPluginName = Constants.CURRENCY_PLUGIN_NAMES[currencyCode]
+  const currencyPlugin = account.currencyConfig[currencyPluginName]
+  try {
+    // $FlowFixMe
+    const supportedCurrencies = currencyPlugin.otherMethods.getActivationSupportedCurrencies()
```
Comment (ahujsak) [inline src/actions/CreateWalletActions.js]: I thought we had said flow fix me's killed flow for the rest of the page. What were the errors you were getting, and can we figure out how to fix.

Comment (kylanhurt) [reply src/actions/CreateWalletActions.js]: The `otherMethods` was not officially part of the API when I wrote this. Let me check with William to see if it's officially in the core / plugins now.

File: src/reducers/scenes/CreateWalletReducer.js [hunk 2]
```diff
@@ -27,6 +46,86 @@ const isCreatingWallet = (state = false, action: Action): boolean => {
...
 }
 
+const isCheckingHandleAvailability = (state = false, action: Action): boolean => {
+  switch (action.type) {
+    case 'IS_CHECKING_HANDLE_AVAILABILITY': {
+      if (action.data === true) {
+        return true
+      } else {
+        return false
+      }
```
Comment (ahujsak) [inline src/reducers/scenes/CreateWalletReducer.js]: = action.data.. don't evaluate.

File: src/reducers/scenes/CreateWalletReducer.js [hunk 3]
```diff
@@ -27,6 +46,86 @@ const isCreatingWallet = (state = false, action: Action): boolean => {
...
+      if (action.data === true) {
+        return false
+      }
+      return state
+    }
+    case 'IS_HANDLE_AVAILABLE': {
+      if (action.data === true) {
+        return true
+      } else {
+        return false
```
Comment (ahujsak) [inline src/reducers/scenes/CreateWalletReducer.js]: return action.data

**Pull request merged**